tools/docker/compose.md

```diff
@@ -6,8 +6,9 @@ order: 5
 
 # Docker Compose
 
-[Docker Compose](https://docs.docker.com/compose/) is a tool to orchestrate Docker containers using a simple YAML file which describes your whole setup.
+[Docker Compose](https://docs.docker.com/compose/) is a tool to orchestrate Docker one or more containers using a simple YAML file which describes your whole setup.
```

Comment:
"orchestrate Docker one or more" does not seem to be in correct order

Would the following work here?
```suggestion
[Docker Compose](https://docs.docker.com/compose/) is a tool to orchestrate one or more Docker containers using a simple YAML file which describes your whole setup.
```

Not sure whether docker-compose works with podman. If it's sensible to say that it works in general with containers, then we could throw out the "Docker" word.

Reply:
Thanks.  docker-compose will work with podman if podman-docker is installed.  I recommend 

`..to orchestrate one or more containers using a simple YAML file ...`

thank you